test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -8,6 +8,13 @@ Describe "Job Cmdlet Tests" -Tag "CI" {
         AfterEach {
             Get-Job | Remove-Job -Force
         }
+
+        It "should not throw exception when -LP alias for LiteralPath is used " {
```

Comment:
Please remove all alias tests - we have already special tests for aliases and no need to add them here.